submissions/OlegPopovych/html_css_popup/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+	<link rel="stylesheet" href="css/main.css">
+	<link rel="shortcut icon" type="image/png" href="img/favicon.png">
+	<title>Popup</title>
+</head>
+
+<body>
+	<nav class="nav">
+		<ul class="nav__list">
+			<li class=" nav__list--item">
+				<h1 class="nav__title">Hello Kottans!</h1>
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Mail</span>
+				</a>
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Photos</span>
+				</a>
+			</li>
+			<li class="nav__list--item relative">
+				<input class="popup__check nav__label nav__input" type="checkbox" id="check-1" />
+				<label class="nav__label " for="check-1">
+					<a class="icon" alt="Menu opener">
```

Comment:
Please, add :focus state to menu icon, when navigate with keyboard

Reply:
I added attribute tabindex="0", but I'm not sure that's enough because the target when we use the tab is `input` in line 30 and the next target is  `a` in line 32.

Reply:
You can add `:focus` state in css for this element,  tabindex="0" is not necessary. Just use .element-classname:focus{ some rules }